enhancements/vmaas/README.md

```diff
@@ -0,0 +1,472 @@
+---
+title: Virtualization as a service
+authors:
+  - Adrien Gentil
+creation-date: 2025-09-15
+last-updated: 2025-09-15
+tracking-link:
+  - TBD
+see-also:
+replaces:
+superseded-by:
+---
+
+# Virtualization as a service
+
+
+## Summary
+
+We want to provide a service that allows a tenant to create and manage VMs.
+
+## Motivation
+
+TBD
+
+### User Stories
+
+#### As a tenant
```

Comment:
It would help to add some prioritization here. Maybe two categories: an MVP that just entails the basics of self-service provisioning out of a catalog that's curated by the service provider, and then everything else.

Also perhaps we should group the networking stories as a category that we know is going to mostly be implemented elsewhere, but we need to make sure VMaaS is using those capabilities.